index.html

```diff
@@ -1,126 +1,136 @@
+          <p>
+            Седона — не аттракцион для туристов, там течёт своя жизнь
+          <ul class="adviсes-list">
+            <li class="adviсes-item"> Жильё
+              <p>Рекомендуем пожить в настоящем мотеле, всё как в кино!</p>
+            </li>
+            <li class="advices-item">Еда
+              <p>Всегда заказывайте топовый фирменный бургер, вы не разочаруетесь!</p>
+            </li>
+            <li class="advices-item">Сувениры
+              <p>Не только китайского, но и настоящего местного производства!</p>
+            </li>
+          </ul>
+          </p>
+        </li>
+          <li class="advantages-item">
+            <h2>Там есть мост дьявола</h2>
+          <p>Да, по нему можно пройти! Если вы осмелитесь, разумеется</p>
+        </li>
+        <li class="advantages-item">
+          <h2>Небольшая площадь</h2>
+          <p>Все достопримечательности находятся очень близко</p>
+        </li>
+        <li class="advantages-item">
+          <h2>Красивая дорога</h2>
+          <p>Ехать в Седону из Лас-Вегаса совсем не скучно!</p>
+        </li>
+        <li class="advantages-item">
+          <h2>Мало туристов</h2>
+          <p>Большинство едет в Гранд Каньон и толпится там</p>
+        </li>
+      </ol>
+    </section>
 
-    <footer class="page-footer">
-      <section class="footer-social">
-        <ul class="footer-social-list">
-          <li class="footer-social-item">
-            <a class="button" href="https://twitter.com/htmlacademy_ru">
-              <span class="visually-hidden">Twitter</span>
-            </a>
-          </li>
-          <li class="footer-social-item">
-            <a class="button" href="https://www.facebook.com/htmlacademy">
-              <span class="visually-hidden">Facebook</span>
-            </a>
-          </li>
-          <li class="footer-social-item">
-            <a class="button" href="https://www.instagram.com/htmlacademy/">
-              <span class="visually-hidden">Instagram</span>
-            </a>
-          </li>
-        </ul>
-      </section>
-      <section class="footer-contacts">
-        <a class="footer-contacts-phone" href="tel:+78128121212">+7 (812) 812-12-12</a>
-      </section>
-      <section>
-        <p class="about-text"><a class="htmlacademy-link" href="https://htmlacademy.ru/">.html academy</a></p>
-      </section>
-    </footer>
-  </body>
+    <section class="hotel-search">
+      <h2>Заинтересовались?</h2>
+      <p>Укажите предполагаемые даты поездки, и мы покажем вам лучшие предложения гостиниц в Седоне</p>
+      <a href="#">Поиск гостиницы в Седоне</a>
+    </section>
+
+    <section class="subscription-form">
+      <h2>Подпишитесь на рассылку</h2>
+      <p>Укажите предполагаемые даты поездки, и мы покажем вам лучшие предложения гостиниц в Седоне</p>
+      <form action="#" method="post">
+        <label for="form-email">
+        <input type="email" id="form-email" name="e-mail" placeholder="Ваш e-mail">
+        </label>
```

Comment:
Атрибуты `for` на лейбле и `id` на инпуте нужны тогда, когда их надо связать.
А связывать их нужно тогда, когда они рядом. Когда инпут вложен в лейбл, в этом уже нет необходимости:
```suggestion
        <label>
          <input type="email" name="e-mail" placeholder="Ваш e-mail">
        </label>
```